google/cloud/storage/client.py

```diff
@@ -836,6 +846,175 @@ def get_hmac_key_metadata(
         metadata.reload(timeout=timeout)  # raises NotFound for missing key
         return metadata
 
+    def generate_signed_post_policy_v4(
+        self,
+        bucket_name,
+        blob_name,
+        conditions,
```

Comment:
`conditions` parameter should be optional.

Reply:
Done